mlibrary/search
# `scss`, `colors`, `utilitiers`, and `src/modules/specialists`.

# Overview
In the continuing efforts of simplifying Search, the focus is on `src/modules/specialists`. Code has been simplified, and the components have been reduced to one: `Specialists` (originally named `SpecialistList`).

## Anything else?
The remaining `.scss` files have been deleted.

`colors` and `utilities` stylesheets have been created to help create more global styles. They have been applied where appropriate.

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Look around the site and see if anything is broken.
